src/App.tsx

```diff
@@ -1,8 +1,20 @@
-import React from 'react';
+import { useState } from 'react';
+import classNames from 'classnames';
 import 'bulma/css/bulma.css';
 import './App.scss';
 
-export const goodsFromServer = [
+enum SortType {
+  None = '',
```

Comment:
The `SortType.None` is set to an empty string. While this works, it might be more intuitive to use a more descriptive default value, such as `None = 'none'`, to avoid confusion when checking for a non-existent sort type.